modules/web/i18n/zh-Hans/messages.zh-Hans.xlf

```diff
@@ -340,15 +340,15 @@
       </trans-unit>
       <trans-unit id="2027381191177847167" datatype="html">
         <source>Exec Commands</source>
-        <target state="new">Exec Commands</target>
+        <target state="new">执行命令</target>
```

Comment:
```suggestion
        <target>执行命令</target>
```

Please remove `state="new"` for translated places.

Reply:
Do I need to remove all `state="new"` in this file?

Reply:
If the line has already been translated and does not require modification,please remove the `state="new"`,If you have enough time, you can do some cleanup for this file.

Reply:
Well received. Have checked all lines with `status="new"`